recipes/PartSeg/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set name = "PartSeg" %}
+{% set version = "0.13.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PartSeg-{{ version }}.tar.gz
+  sha256: f5619cfb1989635026b824110655fa7427de706d7232d183958ab94fb8108503
+
+build:
+  skip: true  # [py<38]
+  entry_points:
+    - PartSeg = PartSeg.launcher_main:main
+    - Tester = PartSeg.test_widget_main:main
```

```suggestion
    - PartSeg-Tester = PartSeg.test_widget_main:main
```

@Czaki - is it ok if we rename this script? `Tester` is a bit too generic and I am concerned it might clash other tools in either conda-forge or the broader ecosystem.

---

Delete it. I will also do this in PartSeg